app/controllers/api/v1/search_results_controller.rb

```diff
@@ -63,7 +65,8 @@ def upload_form_params
       end
 
       def filter_params
-        params.fetch(:filter, {}).permit(:url_equals, :adwords_url_contains)
+        params.fetch(:filter, {}).permit(:url_equals, :adwords_url_contains, :url_contains,
```

Comment:
Thsi alignment is weird 🙈 Either all attributes should be on one one line or split onto its own line:

```ruby
params
   .fetch(:filter, {})
   .permit(:url_equals, :adwords_url_contains, :url_contains, :match_count)
```

Or:

```ruby
params .fetch(:filter, {}).permit(:url_equals, 
                                                     :adwords_url_contains, 
                                                     :url_contains, 
                                                     :match_count)
```